### PR TITLE
schedule lightning talk on OpenStack Image Manager

### PR DESCRIPTION
**Pull request by fkr**
as discussed in todays (2023-05-17) Team IaaS call

### CODE REVIEW

File: lightning_talks.yml [hunk 1]
```diff
@@ -49,3 +49,10 @@ events:
       collaboration mode with ~10 participants in Spain and ~4 online.
       Matej, Roman, and Max will report from the event, present the results and show a demo of the tool.
     location: "https://conf.scs.koeln:8443/SCS-Tech"
+  - summary: "OpenStack Image Manager in Action"
+    begin: 2023-05-11 15:45:00         # uses default timezone above
```
Comment (berendt) [inline lightning_talks.yml:53]: 2023-06-11?

**Review by berendt — CHANGES_REQUESTED** [review]

File: lightning_talks.yml (hunk 1 above)
Comment (fkr) [reply lightning_talks.yml]: doh! fixed :)

**Comment by FriederikeZelke** [comment]
Hey, are you sure with the date? 2023-06-11 is a Sunday.

**Comment by fkr** [comment]
> Hey, are you sure with the date? 2023-06-11 is a Sunday.

perfect - everyone will have time then ;)
Of course, the 1st of june was meant.

**Review by berendt — APPROVED** [review]

**Review by FriederikeZelke — COMMENTED** [review]
:-)

**Pull request merged**